plugins/orchestrator/src/components/next/ExecuteWorkflowPage.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useAsync } from 'react-use';
+
+import { InfoCard, Progress } from '@backstage/core-components';
+import {
+  useApi,
+  useRouteRef,
+  useRouteRefParams,
+} from '@backstage/core-plugin-api';
+import { JsonValue } from '@backstage/types';
+
+import { Button, Grid, Typography } from '@material-ui/core';
+
+import { WorkflowDataInputSchemaResponse } from '@janus-idp/backstage-plugin-orchestrator-common';
+
+import { orchestratorApiRef } from '../../api';
+import {
+  executeWorkflowRouteRef,
+  workflowInstanceRouteRef,
+} from '../../routes';
+import { BaseOrchestratorPage } from './BaseOrchestratorPage';
+
+export interface ExecuteWorkflowPageProps {
+  initialState?: Record<string, JsonValue>;
+}
+
+export const ExecuteWorkflowPage = (props: ExecuteWorkflowPageProps) => {
+  const orchestratorApi = useApi(orchestratorApiRef);
+  const { workflowId } = useRouteRefParams(executeWorkflowRouteRef);
+  const [isExecuting, setIsExecuting] = useState(false);
+
+  // @ts-ignore:
```

Comment:
Why??? ts-ignore
Do you need guidance annotating the type?

Reply:
ack

Reply:
Oh, actually it complains about the `setFormState` for not being used.
You can simply omit it: `const [formState] = useState(...)`.

Reply:
Done